tests/test_tcp.py

```diff
@@ -2609,14 +2609,18 @@ async def client(addr):
 
     def test_remote_shutdown_receives_trailing_data(self):
         if self.implementation == 'asyncio':
+            # this is an issue in asyncio
```

Comment:
Is there a bpo for it, btw?

Reply:
Hmm looks like [BPO-39951](https://bugs.python.org/issue39951) is a close shot, I'm trying that gist on uvloop.